<launch>
  <node name="dvs_txt_node" pkg="dvs_txt" type="dvs_txt" output="screen">
  <remap from="/left_events" to="/event_out_left/slave"/>
  <remap from="/right_events" to="/event_out_right/master"/>

  <param name="file_is_txt" type="bool" value="false" />
  <param name="camera_height" type="int" value="720" />
  <param name="camera_width" type="int" value="1280" />
  <param name="rectify" type="bool" value="true" />

  <param name="file_path" type="string" value="/home/asus/thesis_ws/src/dvs_txt/resource/walking2.txt" />
  <param name="left_csv_file_path" type="string" value="/home/asus/thesis_ws/src/dvs_txt/resource/master.csv" />
  <param name="right_csv_file_path" type="string" value="/home/asus/thesis_ws/src/dvs_txt/resource/slave.csv" />

  <param name="loop_rate" type="double" value="0.04" />

  <param name="disparity_range" type="int" value="128" />

  <param name="nearest_neighbour" type="bool" value="false" />
  <param name="NN_min_num_of_events" type="int" value="2" />
  <param name="NN_block_size" type="int" value="7" />
  
  <param name="publish_slice" type="bool" value="true" />
  <param name="slice_start_time" type="double" value="3.90" />
  <param name="slice_end_time" type="double" value="3.95" />

  <param name="do_adaptive_window" type="bool" value="true" />
  <param name="small_block_size" type="int" value="5" />
  <param name="medium_block_size" type="int" value="11" />
  <param name="large_block_size" type="int" value="13" />
  <param name="threshold_edge" type="int" value="410" />
  </node>

</launch>
